cli/src/main/java/com/box/l10n/mojito/cli/command/DropImportCommand.java

```diff
@@ -147,15 +157,24 @@ private Boolean getImportedFilter() {
      * @throws CommandException if the input doesn't match a number from the map
      * of available {@link Drop}s
      */
-    private Long getSelectedDropId(Map<Long, Drop> numberedAvailableDrops) throws CommandException {
+    private List<Long> getSelectedDropIds(Map<Long, Drop> numberedAvailableDrops) throws CommandException {
+        if (importFetchedParam) {
+            return numberedAvailableDrops.entrySet().stream()
+                    .filter(x -> !Boolean.TRUE.equals(x.getValue().getCanceled()) && x.getValue().getLastImportedDate() == null)
```

Comment:
I don't remember so just asking. if not using `-all` the already imported won't be shown right? And you may want to re-import the drop. So, s hould `x.getValue().getLastImportedDate() == null` be removed?

Reply:
Make sense, especially that the all parameter exists.